### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by Krihag**
Created a read me file

### CODE REVIEW

**Comment by MartineMidttveit** [comment]
Very good.

**Review by MartineMidttveit — CHANGES_REQUESTED** [review]
Can you make it a bit longer?

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This is a read me file
```
Comment (MartineMidttveit) [inline README.md:1]: Bla bla bla

File: README.md
Comment (Krihag) [inline README.md:1]: test

**Pull request closed without merging**